stdlib/public/Concurrency/TaskGroup.swift

```diff
@@ -372,11 +342,13 @@ public struct TaskGroup<ChildTaskResult: Sendable> {
     while let _ = await next() {}
   }
   
-  /// Query whether the group has any remaining tasks.
+  /// A Boolean value that indicates whether the group has any remaining tasks.
   ///
-  /// Task groups are always empty upon entry to the `withTaskGroup` body, and
-  /// become empty again when `withTaskGroup` returns (either by awaiting on all
-  /// pending tasks or cancelling them).
+  /// At the start of the body of a `withTaskGroup(of:returning:body:)` call,
+  /// the task group is always empty.
+  /// It is guaranteed to be empty when returning from that body,
+  /// either because all child tasks have completed
+  /// or because they've been canceled.
```

Comment:
```suggestion
  /// .
```

today's impl always forcefully awaits and that's correct